web/styles/message_row.css

```diff
@@ -126,7 +126,9 @@ $time_column_max_width: 150px;
         padding-left: 10px;
         grid-template:
             25px repeat(3, auto) /
-            $avatar_column_width auto 55px fit-content($time_column_max_width);
+            $avatar_column_width auto max-content fit-content(
+                $time_column_max_width
+            );
```

Comment:
We need to use a fixed width here for these reasons:
* User sent messages have different control icons compared to other messages which results in different width of message controls.
* Message controls icons can change over time. Like the edit icon being replaced by view source icon.
* Locally echoed message have only 2 / 1 icon vs 3 icons after it is sent result in different widths.

Reply:
Removed this commit. I think we could probably get away with `minmax(max-content, 55px)` and keep this intrinsic, but that's a discussion for another day and another PR.